app/locales/ru/locationTracking.json

```diff
@@ -0,0 +1,13 @@
+{
+    "start_logging":"НАЧАТЬ ЗАПИСЬ МЕСТОПОЛОЖЕНИЙ",
```

Comment:
it looks like `start_logging` was renamed to `home_start_tracking` with the V2 redesign. you might want to check out latest code from develop branch. Some other translations variables might be affected.

Reply:
ok will do... thank you